tracing-journald/src/lib.rs

```diff
@@ -339,18 +363,107 @@ impl Visit for EventVisitor<'_> {
     }
 }
 
-fn put_priority(buf: &mut Vec<u8>, meta: &Metadata) {
-    put_field_wellformed(
-        buf,
-        "PRIORITY",
-        match *meta.level() {
-            Level::ERROR => b"3",
-            Level::WARN => b"4",
-            Level::INFO => b"5",
-            Level::DEBUG => b"6",
-            Level::TRACE => b"7",
-        },
-    );
+// Descriptions and examples taken from the Archlinux wiki:
+// https://wiki.archlinux.org/title/Systemd/Journal#Priority_level
+/// A priority (in syslog called severity code) is used to mark the
```

Comment:
We should cite where these are authoritatively defined.

Reply:
The Archlinux wiki links to the [syslog RFC](https://www.rfc-editor.org/rfc/rfc5424#section-6.2.1), which systemdb [includes directly](https://github.com/systemd/systemd/blob/c8428d2d0612176c6db730dee17e9ef92582bb0f/src/basic/log.h#L8). Is it good enough?

Reply:
I think the RFC is an excellent cite, but we should include it in the published docs, not just an internal note.

Reply:
Added both sources for completeness.